docs/docs-content/clusters/edge/site-deployment/site-installation/cluster-deployment.md

```diff
@@ -14,8 +14,13 @@ To complete the Edge Installation process, an Edge host must become a member of
 Select the workflow that best fits your needs.
 
 - [Create an Edge Native Host Cluster](#create-an-edge-native-host-cluster)
-
+  - [Prerequisites](#prerequisites)
+  - [Create Cluster](#create-cluster)
+  - [Validate](#validate)
 - [Add an Edge Host to a Host Cluster](#add-an-edge-host-to-a-host-cluster)
+  - [Prerequisites](#prerequisites-1)
+  - [Add Edge Host to Node Pool](#add-edge-host-to-node-pool)
+  - [Validate](#validate-1)
```

Comment:
We have a TOC on the right-hand side nav menu. We don't need this, nor is this page large enough to warrant one.

Reply:
I am not really sure how this happened - I never added these myself but they just appeared so I thought it was some Docusarus thing that we did. Now I wonder if it came from the Markdown extension that I installed.  I'll delete this.